routers/web/repo/view.go

```diff
@@ -713,14 +713,7 @@ func checkCitationFile(ctx *context.Context, entry *git.TreeEntry) {
 		if entry.Name() == "CITATION.cff" || entry.Name() == "CITATION.bib" {
 			ctx.Data["CitiationExist"] = true
 			// Read Citation file contents
-			blob := entry.Blob()
-			dataRc, err := blob.DataAsync()
-			if err != nil {
-				ctx.ServerError("DataAsync", err)
-				return
-			}
-			defer dataRc.Close()
-			ctx.PageData["citationFileContent"], err = blob.GetBlobContent(setting.UI.MaxDisplayFileSize)
+			ctx.PageData["citationFileContent"], err = entry.Blob().GetBlobContent(setting.UI.MaxDisplayFileSize)
 			if err != nil {
```

Comment:
Could it just ignore this error to prevent from users seeing unnecessary 500 pages?

I do not think reporting such error to end users really makes sense.

Reply:
Done.